Cooper/ClientApp/src.v2/app/components/login/login.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
```

Comment:
**app-** prefix is too general and in common using. Usually it is short form from project name. For example, in PrimeNg p- (p-button), in Angular Material mat- (mat-button). So, we can have **coop-** (**coop**er),  or at all **cooper-**